app/webutil.js

```diff
@@ -146,7 +146,7 @@ export function writeSetting(name, value) {
     if (window.chrome && window.chrome.storage) {
         window.chrome.storage.sync.set(settings);
     } else {
-        localStorage.setItem(name, value);
+        localStorageWrapper("setItem", [name, value]);
```

Comment:
This became a bit meta. Calling methods by giving their names as a string looks very odd.

It also makes it more difficult to follow, as it is no longer apparent what happens here.

I think some duplication in the name of readability can be acceptable here. You could always add a `logOnce()` helper if you want to encapsulate that magic. Could even make it generic based on the message. ;)

Reply:
Good point, fixed now.